src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/Update-AzureRmRecoveryServicesAsrProtectionDirection.md

```diff
@@ -73,6 +92,18 @@ The **Update-AzureRmRecoveryServicesAsrProtectionDirection** cmdlet updates the
 PS C:\> $currentJob = Update-AzureRmRecoveryServicesAsrProtectionDirection -RecoveryPlan $RP -Direction PrimaryToRecovery
 ```
 
+### Example 2
+```
+PS C:\> $currentJob = Update-AzureRmRecoveryServicesAsrProtectionDirection -AzureToAzure -ProtectionContainerMapping $B2ApcmMapping -LogStorageAccountId $cachceStorageId
+ -ReplicationProtectedItem $rpi
+```
+
```

Comment:
@viverm please add a description for this example and make sure that the first line has a trailing backtick ( ` ) to signal continuation on to the next line for copy-paste scenarios